lib/csv_helper/exporter.rb

```diff
@@ -42,8 +44,15 @@ def set_class_for_export
       version.present? ? Institution.version(version.number) : Institution
     end
 
-    def format(key, value)
-      return "\"#{value}\"" if key == :ope && value.present?
+    def format_value(record, field)
+      value = field['non_hash'] ? record.send(field['column']) : record[field['column']]
```

Comment:
It looks like all of these changes in this file revolve around getting the value of a method call into the `Institutions` export.
I propose a simpler solution:
Revert this entire file, and just change each usage of `record[k]` to `record.public_send(k)`.  I count 3 places.
And you can remove `non_hash: true` up in the `Institution` model.

It should be save to use `public_send` since we are only `send`ing methods hard coded in the models (that is, not `send`ing user input)

Let me know if I'm misunderstanding something.

Reply:
No, that makes sense and works.  Thanks!